api/management/commands/acquiredata.py

```diff
@@ -0,0 +1,10 @@
+import os
+from urllib.request import urlretrieve
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        path = os.path.join('data', 'diseases.tsv')
+        url = 'https://raw.githubusercontent.com/cognoma/cancer-data/master/download/diseases.tsv'
```

Comment:
Do you think we want to allow specification of a specific commit, so the user can optionally apply versioning rather than always latest?